sdk/core/System.ClientModel/tests/client/ModelReaderWriter/Models/ExperimentalModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+
+#if SOURCE_GENERATOR
```

Comment:
is this #if for the namespace necessary?  If you were copying the other files that was because it needed to reference the BasicContext when compiled into the test proj but here it doesn't see necessary?